### DISCUSSION TITLE
Reading SCCM Logs with PowerShell

### DISCUSSION THREAD

**Discussion by giscus** [TsekNet/website#15] [Announcements]
# Reading SCCM Logs with PowerShell

How to write SCCM logs to the Event Log

https://tseknet.com/blog/sccmlogs

**Comment by wbatzle** [comment]
This is great. Very well put together. I was wondering if I could bounce another task for SCCM logs off you. I am trying to grab a value in the CAS.log file under the line property. It's  the <ADSite Name=/> value. So far I can only grab the whole contents of the line instead of the bit I want. I tried XML formatting and a few other things. How do you think I should go about it better?

**Reply by TsekNet** [reply]
Hey @wbatzle,

Without knowing what your `cas.log` file looks like exactly, I can't say for certain, but you could always try Regex to extract the contents you want one you've already identified the line you want.

Something like:

```powershell
$found = $string -match '.*some_string_to_match.*'
if ($found) {
    $spid = $matches[1]
}
```

Let me know if that helps!
